src/Microsoft.OpenApi.OData.Reader/Operation/EdmActionOperationHandler.cs

```diff
@@ -37,10 +37,28 @@ protected override void SetBasicInfo(OpenApiOperation operation)
         /// <inheritdoc/>
         protected override void SetRequestBody(OpenApiOperation operation)
         {
-            IEdmAction action = EdmOperation as IEdmAction;
-            if (action != null)
+            if (EdmOperation is IEdmAction action)
```

Comment:
```suggestion
            if (EdmOperation is IEdmAction action && Context.CreateRequestBody(action) is OpenApiRequestBody requestBody)
```

( and delete the block below)